### ISSUE TITLE
Can't show complete java result in Terminal unless resize it

### ISSUE THREAD

**Issue by molly-w** [microsoft/vscode-java-pack#555]
**Environment:**

- VS Code Version: 1.53.2(user setup)
- Extension Version: Java Extension Pack(v0.12.1)
- JDK Type & Version: openjdk version "11.0.8" 
- OS Type & Version: Windows10

**Step to Reproduce:**

hello.txt:
`Hello How are you?`

App.java:
```import java.io.InputStream;
import java.io.FileInputStream;
import java.io.FileNotFoundException;
import java.io.IOException;

public class App {
    public static void main(String args[]) throws FileNotFoundException, IOException {
        InputStream input = new FileInputStream("D:\\absolute path to hello.txt");
        byte[] byteArray = new byte[1024];
        int ArraysRead = input.read(byteArray);
        input.close();
        System.out.println(ArraysRead);
        for (byte i : byteArray) {
            System.out.println ((char) i);
        }      
    }
}
```
**Expected Behavior:**
It should be as the same as the results in external Terminal:
![image](https://user-images.githubusercontent.com/63043224/109900407-4c4ca580-7cd2-11eb-8369-060801c1a48b.png)

**Actual Behavior:**
When running it at the first time in integrated Terminal, it only shows:
![image](https://user-images.githubusercontent.com/63043224/109900523-78682680-7cd2-11eb-9c57-71a3f9f87c31.png)

After enlarging the Terminal and run it again, it showed the result correctly:
![image](https://user-images.githubusercontent.com/63043224/109900578-92096e00-7cd2-11eb-999f-30ff4b9f9344.png)

**Related Issues:** 
Original post: [Problem with Java InputStream's read method not reading the first few bytes](https://stackoverflow.com/questions/66457481/problem-with-java-inputstreams-read-method-not-reading-the-first-few-bytes/66467522#66467522)
Similar issue: [Terminal not showing complete output unless resized](https://github.com/microsoft/vscode/issues/85010)

**Comment by akaroml** [comment]
@testforstephen please take a further look.

**Comment by testforstephen** [comment]
The terminal behavior is controlled by VS Code, not by Java extension. I'm wondering could you try JS to see if it has similar issue?

**Comment by molly-w** [comment]
> The terminal behavior is controlled by VS Code, not by Java extension. I'm wondering could you try JS to see if it has similar issue?

Yes, i'll put an issue to [vscode](https://github.com/microsoft/vscode/issues).
